packages/common/src/utils/moods.ts

```diff
@@ -0,0 +1,27 @@
+export enum Mood {
```

Comment:
@DejayJD  I think this is duped in a few places, for example we have moods exported from the SDK and moods in https://github.com/AudiusProject/audius-protocol/blob/main/packages/mobile/src/utils/moods.ts#L12. Can we just import these from the [SDK](https://github.com/AudiusProject/audius-protocol/blob/main/packages/libs/src/sdk/types/Mood.ts#L26)?

Reply:
Nice! Even better